Algorithm Codes/Graph/AdjacencyMatrixGraph.py

```diff
@@ -2,24 +2,28 @@
 import heapq
 
 class AdjacencyMatrixGraph(Graph):
-    def __init__(self, nodes, edges=None):
+    def __init__(self, nodes, edges=None, is_directed=False):
         self.n = nodes
         self.matrix = [[0 for j in range(self.n)] for i in range(self.n)]
+        self.is_directed = is_directed
 
         if(edges != None):
             for ((u, v), w) in edges:
                 self.matrix[u][v] = w
-                self.matrix[v][u] = w
+                if(not(is_directed)):
+                    self.matrix[v][u] = w
     
     def add_edge(self, edge):
         ((u, v), w) = edge
         self.matrix[u][v] = w
-        self.matrix[v][u] = w
+        if(not(self.is_directed)):
+            self.matrix[v][u] = w
     
     def remove_edge(self, edge):
         (u, v) = edge
         self.matrix[u][v] = 0
-        self.matrix[v][u] = 0
+        if(not(self.is_directed)):

```

Comment:
Python, I think we can use this instead :) 

```
if not self.is_directed:
```

Reply:
Sure.

Reply:
I think after committing this change, we will be good to merge! Thank you so much! :+1:

Reply:
ok, I'll also fix the thing with the runtime, I think that is worth it because you can still achieve the runtime despite using adjacency matrix